Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8198.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.RefreshView)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 8198, "ScrollView at CollectionView at RefreshView always leads to Pull-To-Refresh", PlatformAffected.Android)]
+	public class Issue8198 : TestContentPage
+	{
+		RefreshView _refreshView;
+		Command _refreshCommand;
+  
+		protected override void Init()
+		{
+			Title = "Issue 8198";
+
+			var layout = new StackLayout();
+
+			var instructions = new Label
+			{
+				BackgroundColor = Color.Black,
+				TextColor = Color.White,
+				Text = " Scroll the CollectionView to end and try to scroll up again. If the Refresh Indicator does not appear until it reaches the top, the test has passed."
```

Comment:
> " Scroll the CollectionView to end and try to scroll up again.

I think it'd be useful in this description to say

" Scroll the CollectionView to end, **lift finger off screen**, and then try to scroll up again." 

The behavior doesn't really break if you maintain contact the whole time

Reply:
You're right. I have updated the instructions.